stm32-modules/include/thermocycler-gen2/thermocycler-gen2/eeprom.hpp

```diff
@@ -118,12 +130,17 @@ class Eeprom {
         CONST_B = 0,  // Value of the B constant
         CONST_C = 1,  // Value of the C constant
         // Flag indicating whether constants have been written.
-        // Set to 1 to mark that constants are written with no checksum
-        CONST_FLAG = 2
+        // See \ref EEPROMFlag
+        CONST_FLAG = 2,
+        CONST_A = 3,  // Value of the A constant
     };
 
     // Enumeration of the EEPROM_CONST_FLAG values
-    enum class EEPROMFlag { WRITTEN_NO_CHECKSUM = 1, INVALID = 0xFF };
+    enum class EEPROMFlag {
+        WRITTEN_NO_A = 1,    // Values of B and C are written
+        WRITTEN_WITH_A = 2,  // Values of all constants are written
```

Comment:
you are a hero for trying to keep this backward compatible across proto versions but maybe we should also consider just considering eeproms without A invalid?